bftengine/src/bftengine/PersistentStorageImp.hpp

```diff
@@ -64,6 +64,7 @@ enum ConstMetadataParameterIds : uint32_t {
   LAST_VIEW_TRANSFERRED_SEQ_NUM = 6,
   LAST_STABLE_SEQ_NUM = 7,
   ERASE_METADATA_ON_STARTUP = 9,
+  USER_DATA = 10,
```

Comment:
USER_DATA -> USER_SPECIFIC_DATA?

Reply:
I prefer the shorter one, but I don't have a strong opinion. I can change if you feel strongly about it.